### DISCUSSION TITLE
Disable admin page from netbox via kubernetes ingress

### DISCUSSION THREAD

**Discussion by neharaina1771** [netbox-community/netbox#15097] [Help Wanted!]
Hi Team,

When I am trying to disable /admin page from ingress and wanted this page to redirect to 404 Disabled.It is not working.
Can anyone please help

**Comment by candlerb** [comment]
To disable access to the admin page in Netbox, turn off the "Is Staff" flag from the user(s) in question. That's exactly what it's there to control. You can also do this via [LDAP groups](https://docs.netbox.dev/en/stable/installation/6-ldap/#user-groups-for-permissions). In Netbox 4, this section will be gone completely in any case.

Questions about how to configure Kubernetes ingress controllers would be best asked in a Kubernetes forum, as this is not part of Netbox. Once you've found a suitable forum, I suspect they will want to know which ingress controller you are using and to see the non-working configuration, before they will be able to suggest how to correct it.

See:
* https://www.catb.org/~esr/faqs/smart-questions.html#forum
* https://www.catb.org/~esr/faqs/smart-questions.html#beprecise

**Reply by neharaina1771** [reply]
Thanks for the help .We already turned off the Is Staff but that just restricts the user to login to admin page but we completely want to redirect the admin page to some error page with message like  page disabled.

**Comment by jeremystretch** [comment]
FYI the upcoming v4.0 release will disable the Django admin UI by default (see #12325).

**Reply by neharaina1771** [reply]
Where to set this parameter DJANGO_ADMIN_ENABLED
